### PR TITLE
adding trials page

### PR DESCRIPTION
**Pull request by rubyimrie**
Main change in this page is change to use the slider from vue form as the timeline, and printing the dates underneath.

### CODE REVIEW

**Comment by rubyimrie** [comment]
have updated to have the new icon style and hidden text meaning this will need merged before it looks as it will

File: pages/trial-locations.vue [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+  layout: "default",
+});
+
+export default {
+  components: { Slider },
+  data: () => ({
+    sparqlUrl: "https://query.wikidata.org/sparql",
+    url: "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
+    attribution:
+      'Map data © <a href="https://www.openstreetmap.org/copyright">OpenStreetMap contributors</a>. Historical Maps Layer, 1919-1947 from the <a href="http://maps.nls.uk/projects/api/">NLS Maps API</a>',
```
Comment (Sephster) [inline pages/trial-locations.vue]: I think the copyright symbol ehre needs to be url encoded. I think the code is &copy; from memory

File: pages/trial-locations.vue [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+            trial.year <= this.sliderYears[this.numberRangeValue[1]],
+        );
+      });
+
+      this.markers = markers;
+    },
+    showPageInfo() {
+      Swal.fire({
+        title: "Trial Location Map",
+        html: '<div>This map indicates the location of trial for the accused witches. There is an option to change the year, to show how the numbers of trials changed with time. A few people had <b class="font-bold">multiple trials</b>, which could have been held in <b class="font-bold">different locations</b>. There are <b class="font-bold">3211</b> recorded trials which have been related to witchcraft within the database. However, there are geographical locations noted for only <b class="font-bold">432</b> trials, meaning that many trial locations were not recorded in the surviving documents.</div>',
```
Comment (Sephster) [inline pages/trial-locations.vue]:
```suggestion
        html: '<div>This map indicates the location of trial for the accused witches. There is an option to change the year, to show how the numbers of trials changed with time. A few people had <strong>multiple trials</strong>, which could have been held in <strong>different locations</strong>. There are <b class="font-bold">3211</b> recorded trials which have been related to witchcraft within the database. However, there are geographical locations noted for only <strong>432</strong> trials, meaning that many trial locations were not recorded in the surviving documents.</div>',
```

**Review by Sephster — CHANGES_REQUESTED** [review]

File: pages/trial-locations.vue (hunk 1 above)
Comment (rubyimrie) [reply pages/trial-locations.vue]: done and also changed in other locations used

File: pages/trial-locations.vue (hunk 2 above)
Comment (rubyimrie) [reply pages/trial-locations.vue]: done

File: pages/trial-locations.vue (hunk 1 above)
Comment (Sephster) [reply pages/trial-locations.vue]: Brilliant, thank you

File: pages/memorials.vue [hunk 3]
```diff
@@ -0,0 +1,422 @@
+<template>
```
Comment (Sephster) [inline pages/memorials.vue]: This page seems to have been added in accidentally. I don't remember it being there before. Are you able to remove this as I assume it will be added in a different PR. Thanks Ruby

File: pages/trial-locations.vue (hunk 2 above)
Comment (Sephster) [reply pages/trial-locations.vue]: thank you

**Review by Sephster — CHANGES_REQUESTED** [review]

File: pages/memorials.vue (hunk 3 above)
Comment (rubyimrie) [reply pages/memorials.vue]: oops, done!

Comment (Sephster) [reply pages/memorials.vue]: Looks good now. Thanks Ruby

**Review by Sephster — APPROVED** [review]
